manifests/charts/istiod-remote/files/injection-template.yaml

```diff
@@ -371,8 +371,8 @@ spec:
     volumeMounts:
     - name: workload-socket
       mountPath: /var/run/secrets/workload-spiffe-uds
-    - name: credential-socket
-      mountPath: /var/run/secrets/credential-uds
+    - name: gateway-cert-socket
```

Comment:
Why is it called "gateway"? This is a sidecar

Reply:
make sense, will `external-cert-socket` be more approriate?